app/database/operator/server_data_operator/handlers/post.ts

```diff
@@ -55,6 +54,56 @@ export interface PostHandlerMix {
     handleReceivedPostsInThread: (postsMap: Record<string, Post[]>, prepareRecordsOnly?: boolean) => Promise<PostsInThreadModel[]>;
 }
 
+const mergePostInChannelChunks = async (newChunk: PostsInChannelModel, existingChunks: PostsInChannelModel[]) => {
```

Comment:
Moved this out for easier testing

Reply:
it looks like the logic here has changed a bit which is fine, but I'm wondering if the same logic is applied for android and ios push notifications?